### ISSUE TITLE
Many packages ?

### ISSUE THREAD

**Issue by vindarel** [rongarret/ergolib#2]
Hi, many stuff here look valuable. Don't you want to add them in quicklisp, so than we can easily include them in our projects, and give these idioms a chance to spread ?

**Comment by rongarret** [comment]
Yes, I just haven’t had time to do it.  (I haven’t even had time to read the docs to figure out *how* to do it.)

Also, I really need to make some changes before ergolib is really ready for prime time as a public package.  For example, right now all the code lives in the CL-USER package, which is kind of anti-social.  I do this because that’s convenient for me (and this is my code) but most people would prefer to have it in its own package, which means I need to generate export lists, and that is also time-consuming.

But it’s all open-source so if you want to take a whack at it please feel free to send me pull requests.

**Comment by vindarel** [comment]
cool, this is good news. I see your work similar in scope to [CL21](https://github.com/cl21/cl21/) and I wish they became defaults.

ok I'll dabble with real world CL and I'll feel free to send PRs.

ps: and my 2 cents: IMO [cl-annot](https://github.com/m2ym/cl-annot#annotation-export)'s `export` annotation is very handy and less error-prone than the default to export symbols, because they are next to each other.

**Comment by dkochmanski** [comment]
> ps: and my 2 cents: IMO cl-annot's export annotation is very handy and less error-prone than the default to export symbols, because they are next to each other.

It's easy to write, but harder to read. You can't just navigate to package definition and see, what is package's interface (it's scattered all around the source code).

**Comment by vindarel** [comment]
thanks for the feedback. (I should have spoken in conditional since I'm not very experienced (at all).)

**Comment by rongarret** [comment]
@dkochmanski: IMHO not having a package interface in the source doesn't matter.  You can easily generate a list of exported symbols once the code is loaded.  Also, the interface generally needs to be documented more extensively than simply having a list of symbols.
